src/sentry/profiles/task.py

```diff
@@ -99,15 +108,19 @@ def _get_profiles_producer_from_topic(topic: Topic) -> KafkaProducer:
 )
 def process_profile_task(
     profile: Profile | None = None,
-    payload: Any = None,
+    payload: str | bytes | None = None,
     sampled: bool = True,
     **kwargs: Any,
 ) -> None:
     if not sampled and not options.get("profiling.profile_metrics.unsampled_profiles.enabled"):
         return
 
     if payload:
-        message_dict = msgpack.unpackb(payload, use_list=False)
+        if isinstance(payload, str):  # It's been b64encoded for taskworker
+            message_dict = decode_payload(payload)
+        else:
+            message_dict = msgpack.unpackb(payload, use_list=False)
```

Comment:
Shouldn't this have already gone out and deployed?

Reply:
Some weird GH bug, fixed now.